### PR TITLE
Sync main

### PR DESCRIPTION
**Pull request by JingLi1998**
No clue if I'm doing this right 😓 

There were similar changes between our fork and master (please double check if it is okay to discard them)

### CODE REVIEW

File: packages/relay-runtime/util/RelayConcreteNode.js [hunk 1]
```diff
@@ -47,16 +47,26 @@ export type ProvidedVariablesType = {+[key: string]: {get(): mixed}};
...
-  +cacheID?: string,
-  +id: string | null,
-  +text: string | null,
-  // common fields
-  +name: string,
-  +operationKind: 'mutation' | 'query' | 'subscription',
-  +providedVariables?: ProvidedVariablesType,
-  +metadata: {[key: string]: mixed, ...},
-};
+export type RequestParameters =
```
Comment (JingLi1998) [inline packages/relay-runtime/util/RelayConcreteNode.js:50]: @JingLi1998 resolved a merge conflict here

File: compiler/crates/relay-compiler/src/artifact_content/content.rs [hunk 2]
```diff
@@ -180,22 +178,18 @@ pub fn generate_operation(
...
-            })),
-        ) if *include_query_text => {
-            request_parameters.id = id_and_text_hash;
-            request_parameters.text = text.clone();
-        }
-        (Some(_), _) => {
-            request_parameters.id = id_and_text_hash;
-        }
-        (None, _) => {
+
```
Comment (JingLi1998) [inline compiler/crates/relay-compiler/src/artifact_content/content.rs:181]: @JingLi1998  resolved a merge conflict here. (Although it looks like something went wrong)

File: compiler/crates/relay-codegen/src/build_ast.rs [hunk 3]
```diff
@@ -438,10 +433,13 @@ impl<'schema, 'builder, 'config> CodegenBuilder<'schema, 'builder, 'config> {
                     ),
                 operation: Primitive::GraphQLModuleDependency(GraphQLModuleDependency::Name(refetch_metadata.operation_name.into())),
             };
-            if let Some(identifier_field) = refetch_metadata.identifier_field {
+            if let Some(identifier_info) = &refetch_metadata.identifier_info {
```
Comment (JingLi1998) [inline compiler/crates/relay-codegen/src/build_ast.rs:436]: @JingLi1998 resolved a merge conflict here

**Comment by tomgasson** [comment]
I've pushed this bump directly

**Pull request closed without merging**